Bigtable/tests/Unit/BigtableClientTest.php

```diff
@@ -33,7 +34,7 @@ class BigtableClientTest extends TestCase
     public function set_up()
     {
         $this->client = TestHelpers::stub(BigtableClient::class, [
-            ['projectId' => 'my-project']
+            ['projectId' => 'my-project', 'credentials' => new InsecureCredentialsWrapper()]
```

Comment:
Are UTs not working without this change? Ideally, unless specifically checked for, they should not need `credentials` since they don't connect hit emulator/prod.
Also, I see very less references of `InsecureCredentialsWrapper`, it seems abstracted out by `EmulatorTrait`.

Reply:
Similarly to my other comment, without this change, we get an error for missing credentials. This is because when the tests are run as one, an application default credential is set to the fixtures by one of the tests, which gets propagated through to this test. But when run individually, this doesn't happen. So *one* solution is doing what I have here, which is to set credentials to the `InsecureCredentialsWrapper`.